File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/CatalogItem.java
package com.itmo.microservices.demo.order.api.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CatalogItem {
    private UUID uuid;
    private String title;
    private String description;
    private int price;
    private int amount;
}

File: src/main/kotlin/com/itmo/microservices/demo/items/api/model/CatalogItem.kt
package com.itmo.microservices.demo.items.api.model

import com.fasterxml.jackson.annotation.JsonInclude
import java.util.*

@JsonInclude(JsonInclude.Include.NON_NULL)
data class CatalogItem(
        val title: String,
        val description: String,
        val price: Int = 100,
        val amount: Int
)
